website/source/layouts/guides.erb

```diff
@@ -65,6 +65,15 @@
         <a href="/guides/namespaces.html">Namespaces</a>
       </li>
 
+      <li<%= sidebar_current("guides-load-balancing") %>>
```

Comment:
Lets place the `Load Balancing` guides section navigation link right above `Web UI`. We moved away alphabetical sorting with the new reorganization.